onnxruntime/test/python/onnxruntime_test_python_iobinding.py

```diff
@@ -24,6 +24,14 @@ def create_expected_output(self):
     def create_expected_output_alternate(self):
         return np.array([[2.0, 8.0], [18.0, 32.0], [50.0, 72.0]], dtype=np.float32)
 
+    def test_ort_value_properties(self):
```

Comment:
If I am not mistaken, such a "generic" (non-IOBinding related) OrtValue test already exists in the regular python test file. This file as the name indicates is only for IOBinding related tests.

Reply:
I could not find it

Reply:
https://github.com/microsoft/onnxruntime/blob/ff655175ff2ecb32f24cdc43870a6c34667d73d6/onnxruntime/test/python/onnxruntime_test_python.py#L836